File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/engine/SharedData.java
/**
 * 
 */
package com.abubusoft.xenon.engine;

/**
 * <p>Rappresenta un dato condiviso tra la fase LOGIC e la fase
 * RENDER. Nello specifico, copia i dati di render</p>
 * 
 * @author Francesco Benincasa
 *
 */
public interface SharedData {

	/**
	 * passa i dati dallo phase LOGIC alla fase RENDER
	 */
	public void update();
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/engine/SharedData.kt
/**
 *
 */
package com.abubusoft.xenon.engine

/**
 *
 * Rappresenta un dato condiviso tra la fase LOGIC e la fase
 * RENDER. Nello specifico, copia i dati di render
 *
 * @author Francesco Benincasa
 */
interface SharedData {
    /**
     * passa i dati dallo phase LOGIC alla fase RENDER
     */
    fun update()
}